### PR TITLE
Add initial documentation for building servo on OpenHarmony

### PR DESCRIPTION
**Pull request by jschwe**

### CODE REVIEW

**Comment by jschwe** [comment]
@mukilan Perhaps you could have a look?

File: src/hacking/building-for-openharmony.md [hunk 1]
```diff
@@ -0,0 +1,120 @@
...
+You can manage installed OpenHarmony SDKs by clicking File->Settings and selecting "OpenHarmony SDK".
+After setting a suitable installation path, you can select the components you want to install for each available API version.
+DevEco Studio will automatically download and install the components for you.
+
+
+#### Manual installation of the OpenHarmony SDK (e.g. on Linux)
+
+<div class="warning _note">
+    Before rushing and downloading the OH SDK from gitee as described here, please note that you will also need `hvigor` to compile applications.
+    `hvigor` is currently only available via the [HarmonyOS NEXT commandline tools], which also contains a copy of the OpenHarmony SDK.
```
Comment (mukilan) [inline src/hacking/building-for-openharmony.md:31]: The markdown syntax doesn't seem to have any effect inside the warning div. Both the backtick quoted text and reference links are not [rendered correctly](https://github.com/servo/book/blob/577a8c416393497d3868065c9c463dce12a85273/src/hacking/building-for-openharmony.md)

File: src/hacking/building-for-openharmony.md [hunk 2]
```diff
@@ -0,0 +1,120 @@
...
+#### Manual installation of hvigor without the commandline tools
+
+<div class="warning _note">
+This section is not fully tested and may change based on user feedback.
+It's recommended to install the commandline-tools bundle. If you decide to install manually, you need to take
+care to install the `hvigor` version matching the requirements of your project.
+</div>
+
+`hvigor` (not the wrapper `hvigorw`) is also available via `npm`. 
+1. Install the same nodejs version as the commandline-tools ship.
```
Comment (mukilan) [inline src/hacking/building-for-openharmony.md:65]: Is there a way for users to figure out the node version? It might be helpful to add that here.

Comment (jschwe) [reply src/hacking/building-for-openharmony.md:65]: I'm not sure. I thought I saw the required version documented somewhere, but I can't find it right now. 
So currently the only approach is "Check what the commandline tools bundle ships".  I did document the version shipped in Harmony OS NEXT though.
I imagine that a lot of the things will get easier after the official release, as more components become freely available / opensourced.

**Review by mukilan — APPROVED** [review]

File: src/hacking/building-for-openharmony.md (hunk 1 above)
Comment (jschwe) [reply src/hacking/building-for-openharmony.md]: Thanks, thats good to know!

**Pull request merged**